Online-vs-Offline-Editors

```diff
@@ -0,0 +1,71 @@
+**Online Offline Editors**
+**There are several offline IDE that programmers use in order to stay more productive to learn and be more productive at work**
```

Comment:
Like I said in the last PR, this is a non-standard term. Define what you mean by these. I have no idea what offline IDE is, or what online IDE is.